yarn.lock

```diff
@@ -23543,6 +23571,16 @@ yup@0.32.6:
     property-expr "^2.0.4"
     toposort "^2.0.2"
 
+yup@^1.3.3:
```

Comment:
Our version of Yup isn't being deduplicated?

Reply:
We're on an older version in force, 0.32. Could upgrade here but not too sure what that would look like in this PR